kubectl-rook-ceph.sh

```diff
@@ -378,50 +378,50 @@ function check_mgr_pods_status_and_count() {
 # 'kubectl rook-ceph debug commands
 ####################################################################################################
 
-function remove_probe_from_deployment(){
+function remove_probe_from_deployment() {
   deployment_spec=$1
-  deployment_spec=$(jq 'del(.template.spec.containers[0].livenessProbe)' <<< $deployment_spec)
-  deployment_spec=$(jq 'del(.template.spec.containers[0].startupProbe)' <<< $deployment_spec)
-  echo "$deployment_spec" # for returning the updated deployment_spec 
+  deployment_spec=$(jq 'del(.template.spec.containers[0].livenessProbe)' <<<$deployment_spec)
+  deployment_spec=$(jq 'del(.template.spec.containers[0].startupProbe)' <<<$deployment_spec)
+  echo "$deployment_spec" # for returning the updated deployment_spec
 }
 
-function update_deployment_spec_command(){
+function update_deployment_spec_command() {
   deployment_spec=$1
   main_command='["sleep","infinity"]'
-  deployment_spec=$(jq .template.spec.containers[0].command="$main_command" <<< $deployment_spec)
-  deployment_spec=$(jq .template.spec.containers[0].args=[] <<< $deployment_spec)
```

Comment:
The editor automatically changed all of these? Seems like this space is good to keep, unless the editor really removes it.

Reply:
yes, it was the editor